src/tools/interop/idt/capture/base.py

```diff
@@ -30,19 +30,32 @@ def __init__(self, artifact_dir: str) -> None:
         """
         raise NotImplementedError
 
+    @abstractmethod
+    async def connect(self) -> None:
+        """
+        Establish connections to log targets for this platform
+        """
+        raise NotImplementedError
+
     @abstractmethod
     async def start_streaming(self) -> None:
         """
         Begin streaming logs
-        Start should be able to be called repeatedly without restarting streaming
```

Comment:
Now that this is removed, does that imply that re-calling start WILL re-start the stream?

Reply:
For the current Android implementation, it does not, and using the included shell utilities class will generally prevent restart. However, I've removed the weird functionality of multiple calls to start and stop of the platform by moving the start and stop calls for platform from the individual ecosystems to the controller in factory instead.

Reply:
I've further cleaned up the shell utility to log this behavior. Will close this out!